lib/spack/spack/test/cmd/env.py

```diff
@@ -823,7 +841,10 @@ def test_env_with_included_config_missing_file(tmpdir, mutable_empty_config):
     spack_yaml = tmpdir.join("spack.yaml")
     missing_file = tmpdir.join("packages.yaml")
     with spack_yaml.open("w") as f:
-        f.write("spack:\n  include:\n    - {0}\n".format(missing_file.strpath))
+        file_seperator = "//"
+        if is_windows:
+          file_seperator = "///"
```

Comment:
I'm not a fan of having to do this all over the place. Is there a helper function to create these file paths?

Reply:
I assume the `///` is an artifact of file urls? IIRC there's no handler now, but pathlib/urllib should make the need for a handler unnecessary if you can use those tools?

Reply:
yup, makes sense. I cleaned it up with pathlib, lmk if it looks better now.